### PR TITLE
SSH server: redesign menu; add to dispatcher

### PR DESCRIPTION
**Pull request by hius07**
Menu standardized to checkbox, an action added to dispatcher, infomessages added/extended.

--

--

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/7765)

### CODE REVIEW

**Comment by hius07** [comment]
May I add saving `allow_no_password` to settings? Many people go online at home, safely.

**Comment by NiLuJe** [comment]
Sure. I'd rather just dump this whole thing into a fire, but, oh, well.

**Comment by hius07** [comment]
The PR is finished.

File: plugins/SSH.koplugin/main.lua [hunk 1]
```diff
@@ -68,15 +70,15 @@ function SSH:start()
...
                     self.SSH_port,
                     Device.retrieveNetworkInfo and Device:retrieveNetworkInfo() or _("Could not retrieve network info.")),
         }
         UIManager:show(info)
     else
         local info = InfoMessage:new{
-                timeout = 5,
-                text = _("Error"),
+                icon = "notice-warning",
+                text = _("Failed to start SSH server"),
```
Comment (Frenzie) [inline plugins/SSH.koplugin/main.lua:81]:
```suggestion
                text = _("Failed to start SSH server."),
```

File: plugins/SSH.koplugin/main.lua [hunk 2]
```diff
@@ -138,54 +153,54 @@ function SSH:addToMainMenu(menu_items)
...
             {
-                text = _("Change SSH port"),
+                text = _("SSH public key"),
                 keep_menu_open = true,
                 enabled_func = function() return not self:isRunning() end,
-                callback = function() return self:show_port_dialog() end,
+                callback = function()
+                    local info = InfoMessage:new{
+                        timeout = 60,
+                        text = T(_("Put your public SSH keys in\n %1"), BD.filepath(path.."/settings/SSH/authorized_keys")),
```
Comment (Frenzie) [inline plugins/SSH.koplugin/main.lua:184]:
```suggestion
                        text = T(_("Put your public SSH keys in\n%1"), BD.filepath(path.."/settings/SSH/authorized_keys")),
```

**Review by Frenzie — COMMENTED** [review]
Cool, couple of nits.

**Review by Frenzie — APPROVED** [review]

**Pull request merged**